ngraph/core/include/openvino/op/ctc_loss.hpp

```diff
@@ -66,6 +66,10 @@ class OPENVINO_API CTCLoss : public Op {
     bool preprocess_collapse_repeated_;
     bool ctc_merge_repeated_;
     bool unique_;
+    template <class T>
```

Comment:
For this ctc_loss, it seems that the `shape_infer` doesn't touch the private field of CTCLoss. So this friend function could be removed.